pkgs/by-name/an/antora-ui-default/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  fetchFromGitLab,
+  lib,
+  stdenvNoCC,
+}:
+let
+  srcFetchFromGitLab = {
+    hash = "sha256-q2FwkwzjanxTIxjMpCyMpzPt782uYZiWVdZ7Eev79oM=";
+    owner = "trueNAHO";
+    repo = "antora-ui-default";
+    rev = "83bf9bf5f22a6dee397f8b089eb0315c14a278b5";
+  };
+in
+stdenvNoCC.mkDerivation {
+  installPhase = ''install --mode 755 -D "$src/ui-bundle.zip" "$out"'';
```

Comment:
The typical "structure" of a derivation from top-to-bottom is pname, version, src, patches, build, check, install, meta.

Just for humans. What you have is fine, but I do like that general structure used in other derivations.